src/utils.ts

```diff
@@ -240,11 +239,11 @@ export function stripTrailingSlash(aPath: string): string {
  * when passing on a failure from a Promise error handler.
  * @param msg - Should be either a string or an Error
  */
-export function errP(msg: string | Error): Promise<never> {
+export function errP(msg?: string | Error): Promise<never> {
     const isErrorLike = (thing: any): thing is Error => !!thing.message;
 
     let e: Error;
-    if (!msg) {
+    if (msg === undefined) {
```

Comment:
use isUndefined?

Reply:
Changed method to only accept strings, and changed to isEmpty